engine/src/iutil.cpp

```diff
@@ -464,8 +464,14 @@ void MCImage::crop(MCRectangle *newrect)
 	MCImageBitmapCopyRegionToBitmap(t_bitmap, t_cropimage, xorigin, yorigin, trect.x, trect.y, trect.width, trect.height);
 
 	unlockbitmap(t_bitmap);
+	
+	// PM-2015-07-13: [[ Bug 15590 ]] Fix memory leak
+	MCImageFreeBitmap(t_bitmap);
 
```

Comment:
The image bitmap pointed to by ```t_bitmap``` shouldn't need to be freed, as it will be freed by ```unlockbitmap```